app/config/ApplicationConfig.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import play.api.Configuration
+
+import javax.inject.Inject
+
+class ApplicationConfig @Inject()(runModeConfiguration: Configuration) {
+
+  lazy val environment: String = runModeConfiguration.get[String]("taas.environment")
```

Comment:
what happens if this is null?

Reply:
I'm not really sure how the original approach would work as the value would need to come from a mock, but the controller needs to be injected.

`lazy val environment: String = runModeConfiguration.getOptional[String]("taas.environment").getOrElse("")`

Went this route instead